AppController/djinn.rb

```diff
@@ -5936,7 +5975,8 @@ def start_xmpp_for_app(app, port, app_language)
       start_cmd = "#{PYTHON27} #{APPSCALE_HOME}/XMPPReceiver/xmpp_receiver.py #{app} #{login_ip} #{@@secret}"
       stop_cmd = "#{PYTHON27} #{APPSCALE_HOME}/scripts/stop_service.py " +
         "xmpp_receiver.py #{app}"
-      MonitInterface.start(watch_name, start_cmd, stop_cmd, 9999)
+      MonitInterface.start(watch_name, start_cmd, stop_cmd, [9999], nil,
```

Comment:
Cassandra doesn't listen on 9999. It's just a stand-in port number that we use for services whose ports we don't care about. I think the xmpp server listens via socket rather than port, so I'm not sure what I would change this to.